src/containers/Login/index.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+
+export class index extends Component {
+  constructor() {
+    super()
+    this.state={
+      email:'',
+      password:''
+    }
+  }
+
+  login = () => {
+    console.log(this.state)
+  }
+
+  handleChange =(e) => {
+    this.setState={[e.etarget.name]: e.target.value}
```

Comment:
Does this work? setState is a method that takes in an object for state but you are doing an assignment like it is a variable.

I feel like it should be
this.setState({[e.target.name]: e.target.value})

Reply:
You're totally right, my bad.

Reply:
Fixed Now!